models/models.js

```diff
@@ -9,4 +9,18 @@ exports.fetchCategories = ()=>{
         return result.rows
     })
 
+}
+
+exports.fetchReviews= () => {
+
+    return db.query(`
+
+    SELECT owner, title,reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comments.review_id)
```

Comment:
Count is a good choice here, maybe have a look at COUNT .. AS - after all, we do want this to be called comment_count